course/grades.py

```diff
@@ -108,6 +108,7 @@ def view_participant_grades(pctx, participation_id=None):
                 "grade_time")
             .select_related("participation")
             .select_related("participation__user")
+            .select_related("flow_session")
```

Comment:
Are these `select_related` still needed, given that we won't be looking at the flow sesions in grade finding?

Reply:
Yes, you're right. Will revert that.

Reply:
However, that is needed at least for `view_grades_by_opportunity`, see [here](https://github.com/inducer/relate/blob/f741e6b87c578c0ba2271ca764fdcfb4c417a2d3/course/templates/course/gradebook-by-opp.html#L171-L210).